{
  "uid" : "3a3aa6847e4962f5",
  "name" : "Test for authorization customer.",
  "fullName" : "tests.login_page_test.TestLoginPage#test_logining_user",
  "historyId" : "8258fdee58153c5b5e4927cdad00af1a",
  "time" : {
    "start" : 1728784173634,
    "stop" : 1728784173634,
    "duration" : 0
  },
  "status" : "broken",
  "statusMessage" : "selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome=129.0.6668.89)\nStacktrace:\n#0 0x55f89f3ad02a <unknown>\n#1 0x55f89f0935e0 <unknown>\n#2 0x55f89f08b7eb <unknown>\n#3 0x55f89f07ba09 <unknown>\n#4 0x55f89f07d744 <unknown>\n#5 0x55f89f07bcbd <unknown>\n#6 0x55f89f07b52f <unknown>\n#7 0x55f89f07b422 <unknown>\n#8 0x55f89f079493 <unknown>\n#9 0x55f89f079ada <unknown>\n#10 0x55f89f095f41 <unknown>\n#11 0x55f89f1278c5 <unknown>\n#12 0x55f89f107b22 <unknown>\n#13 0x55f89f126d7d <unknown>\n#14 0x55f89f1078c3 <unknown>\n#15 0x55f89f0d56b3 <unknown>\n#16 0x55f89f0d668e <unknown>\n#17 0x55f89f377a2b <unknown>\n#18 0x55f89f37b9b1 <unknown>\n#19 0x55f89f364225 <unknown>\n#20 0x55f89f37c532 <unknown>\n#21 0x55f89f34938f <unknown>\n#22 0x55f89f39bf28 <unknown>\n#23 0x55f89f39c0f3 <unknown>\n#24 0x55f89f3abe7c <unknown>\n#25 0x7ff11e294ac3 <unknown>",
  "statusTrace" : "driver = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"1b4aa53da7dc77fcc0cb5406fe20ae78\")>\n\n    @pytest.fixture()\n    @allure.step(f'Open page: {MainPageLinks.URL_MAIN_PAGE}')\n    def main_page_open(driver):\n>       driver.get(MainPageLinks.URL_MAIN_PAGE)\n\ntests/conftest.py:22: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py:353: in get\n    self.execute(Command.GET, {\"url\": url})\n/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py:344: in execute\n    self.error_handler.check_response(response)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.remote.errorhandler.ErrorHandler object at 0x7faf1f7d5d90>\nresponse = {'status': 500, 'value': '{\"value\":{\"error\":\"unknown error\",\"message\":\"unknown error: net::ERR_CONNECTION_REFUSED\\\\n  ...\\\\n#23 0x55f89f39c0f3 \\\\u003Cunknown>\\\\n#24 0x55f89f3abe7c \\\\u003Cunknown>\\\\n#25 0x7ff11e294ac3 \\\\u003Cunknown>\\\\n\"}}'}\n\n    def check_response(self, response: Dict[str, Any]) -> None:\n        \"\"\"Checks that a JSON response from the WebDriver does not have an\n        error.\n    \n        :Args:\n         - response - The JSON response from the WebDriver server as a dictionary\n           object.\n    \n        :Raises: If the response contains an error message.\n        \"\"\"\n        status = response.get(\"status\", None)\n        if not status or status == ErrorCode.SUCCESS:\n            return\n        value = None\n        message = response.get(\"message\", \"\")\n        screen: str = response.get(\"screen\", \"\")\n        stacktrace = None\n        if isinstance(status, int):\n            value_json = response.get(\"value\", None)\n            if value_json and isinstance(value_json, str):\n                import json\n    \n                try:\n                    value = json.loads(value_json)\n                    if len(value) == 1:\n                        value = value[\"value\"]\n                    status = value.get(\"error\", None)\n                    if not status:\n                        status = value.get(\"status\", ErrorCode.UNKNOWN_ERROR)\n                        message = value.get(\"value\") or value.get(\"message\")\n                        if not isinstance(message, str):\n                            value = message\n                            message = message.get(\"message\")\n                    else:\n                        message = value.get(\"message\", None)\n                except ValueError:\n                    pass\n    \n        exception_class: Type[WebDriverException]\n        e = ErrorCode()\n        error_codes = [item for item in dir(e) if not item.startswith(\"__\")]\n        for error_code in error_codes:\n            error_info = getattr(ErrorCode, error_code)\n            if isinstance(error_info, list) and status in error_info:\n                exception_class = getattr(ExceptionMapping, error_code, WebDriverException)\n                break\n        else:\n            exception_class = WebDriverException\n    \n        if not value:\n            value = response[\"value\"]\n        if isinstance(value, str):\n            raise exception_class(value)\n        if message == \"\" and \"message\" in value:\n            message = value[\"message\"]\n    \n        screen = None  # type: ignore[assignment]\n        if \"screen\" in value:\n            screen = value[\"screen\"]\n    \n        stacktrace = None\n        st_value = value.get(\"stackTrace\") or value.get(\"stacktrace\")\n        if st_value:\n            if isinstance(st_value, str):\n                stacktrace = st_value.split(\"\\n\")\n            else:\n                stacktrace = []\n                try:\n                    for frame in st_value:\n                        line = frame.get(\"lineNumber\", \"\")\n                        file = frame.get(\"fileName\", \"<anonymous>\")\n                        if line:\n                            file = f\"{file}:{line}\"\n                        meth = frame.get(\"methodName\", \"<anonymous>\")\n                        if \"className\" in frame:\n                            meth = f\"{frame['className']}.{meth}\"\n                        msg = \"    at %s (%s)\"\n                        msg = msg % (meth, file)\n                        stacktrace.append(msg)\n                except TypeError:\n                    pass\n        if exception_class == UnexpectedAlertPresentException:\n            alert_text = None\n            if \"data\" in value:\n                alert_text = value[\"data\"].get(\"text\")\n            elif \"alert\" in value:\n                alert_text = value[\"alert\"].get(\"text\")\n            raise exception_class(message, screen, stacktrace, alert_text)  # type: ignore[call-arg]  # mypy is not smart enough here\n>       raise exception_class(message, screen, stacktrace)\nE       selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\nE         (Session info: chrome=129.0.6668.89)\nE       Stacktrace:\nE       #0 0x55f89f3ad02a <unknown>\nE       #1 0x55f89f0935e0 <unknown>\nE       #2 0x55f89f08b7eb <unknown>\nE       #3 0x55f89f07ba09 <unknown>\nE       #4 0x55f89f07d744 <unknown>\nE       #5 0x55f89f07bcbd <unknown>\nE       #6 0x55f89f07b52f <unknown>\nE       #7 0x55f89f07b422 <unknown>\nE       #8 0x55f89f079493 <unknown>\nE       #9 0x55f89f079ada <unknown>\nE       #10 0x55f89f095f41 <unknown>\nE       #11 0x55f89f1278c5 <unknown>\nE       #12 0x55f89f107b22 <unknown>\nE       #13 0x55f89f126d7d <unknown>\nE       #14 0x55f89f1078c3 <unknown>\nE       #15 0x55f89f0d56b3 <unknown>\nE       #16 0x55f89f0d668e <unknown>\nE       #17 0x55f89f377a2b <unknown>\nE       #18 0x55f89f37b9b1 <unknown>\nE       #19 0x55f89f364225 <unknown>\nE       #20 0x55f89f37c532 <unknown>\nE       #21 0x55f89f34938f <unknown>\nE       #22 0x55f89f39bf28 <unknown>\nE       #23 0x55f89f39c0f3 <unknown>\nE       #24 0x55f89f3abe7c <unknown>\nE       #25 0x7ff11e294ac3 <unknown>\n\n/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/errorhandler.py:229: WebDriverException",
  "flaky" : false,
  "newFailed" : false,
  "beforeStages" : [ {
    "name" : "clear_allure_results_folder",
    "time" : {
      "start" : 1728784074935,
      "stop" : 1728784074936,
      "duration" : 1
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  }, {
    "name" : "main_page_open",
    "time" : {
      "start" : 1728784174406,
      "stop" : 1728784174824,
      "duration" : 418
    },
    "status" : "broken",
    "statusMessage" : "selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome=129.0.6668.89)\nStacktrace:\n#0 0x55f89f3ad02a <unknown>\n#1 0x55f89f0935e0 <unknown>\n#2 0x55f89f08b7eb <unknown>\n#3 0x55f89f07ba09 <unknown>\n#4 0x55f89f07d744 <unknown>\n#5 0x55f89f07bcbd <unknown>\n#6 0x55f89f07b52f <unknown>\n#7 0x55f89f07b422 <unknown>\n#8 0x55f89f079493 <unknown>\n#9 0x55f89f079ada <unknown>\n#10 0x55f89f095f41 <unknown>\n#11 0x55f89f1278c5 <unknown>\n#12 0x55f89f107b22 <unknown>\n#13 0x55f89f126d7d <unknown>\n#14 0x55f89f1078c3 <unknown>\n#15 0x55f89f0d56b3 <unknown>\n#16 0x55f89f0d668e <unknown>\n#17 0x55f89f377a2b <unknown>\n#18 0x55f89f37b9b1 <unknown>\n#19 0x55f89f364225 <unknown>\n#20 0x55f89f37c532 <unknown>\n#21 0x55f89f34938f <unknown>\n#22 0x55f89f39bf28 <unknown>\n#23 0x55f89f39c0f3 <unknown>\n#24 0x55f89f3abe7c <unknown>\n#25 0x7ff11e294ac3 <unknown>\n\n",
    "statusTrace" : "  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/_pytest/fixtures.py\", line 1123, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/_pytest/fixtures.py\", line 902, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n                     ^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/auto-tests-python/auto-tests-python/tests/conftest.py\", line 22, in main_page_open\n    driver.get(MainPageLinks.URL_MAIN_PAGE)\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py\", line 353, in get\n    self.execute(Command.GET, {\"url\": url})\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py\", line 344, in execute\n    self.error_handler.check_response(response)\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n",
    "steps" : [ {
      "name" : "Open page: https://brainup.site/",
      "time" : {
        "start" : 1728784174406,
        "stop" : 1728784174824,
        "duration" : 418
      },
      "status" : "broken",
      "statusMessage" : "selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome=129.0.6668.89)\nStacktrace:\n#0 0x55f89f3ad02a <unknown>\n#1 0x55f89f0935e0 <unknown>\n#2 0x55f89f08b7eb <unknown>\n#3 0x55f89f07ba09 <unknown>\n#4 0x55f89f07d744 <unknown>\n#5 0x55f89f07bcbd <unknown>\n#6 0x55f89f07b52f <unknown>\n#7 0x55f89f07b422 <unknown>\n#8 0x55f89f079493 <unknown>\n#9 0x55f89f079ada <unknown>\n#10 0x55f89f095f41 <unknown>\n#11 0x55f89f1278c5 <unknown>\n#12 0x55f89f107b22 <unknown>\n#13 0x55f89f126d7d <unknown>\n#14 0x55f89f1078c3 <unknown>\n#15 0x55f89f0d56b3 <unknown>\n#16 0x55f89f0d668e <unknown>\n#17 0x55f89f377a2b <unknown>\n#18 0x55f89f37b9b1 <unknown>\n#19 0x55f89f364225 <unknown>\n#20 0x55f89f37c532 <unknown>\n#21 0x55f89f34938f <unknown>\n#22 0x55f89f39bf28 <unknown>\n#23 0x55f89f39c0f3 <unknown>\n#24 0x55f89f3abe7c <unknown>\n#25 0x7ff11e294ac3 <unknown>\n\n",
      "statusTrace" : "  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/auto-tests-python/auto-tests-python/tests/conftest.py\", line 22, in main_page_open\n    driver.get(MainPageLinks.URL_MAIN_PAGE)\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py\", line 353, in get\n    self.execute(Command.GET, {\"url\": url})\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py\", line 344, in execute\n    self.error_handler.check_response(response)\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ {
        "name" : "driver",
        "value" : "<selenium.webdriver.chrome.webdriver.WebDriver (session=\"1b4aa53da7dc77fcc0cb5406fe20ae78\")>"
      } ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : true,
      "hasContent" : true
    } ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 1,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : true
  }, {
    "name" : "driver",
    "time" : {
      "start" : 1728784173635,
      "stop" : 1728784174406,
      "duration" : 771
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  } ],
  "afterStages" : [ {
    "name" : "driver::0",
    "time" : {
      "start" : 1728784174860,
      "stop" : 1728784174922,
      "duration" : 62
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  } ],
  "labels" : [ {
    "name" : "epic",
    "value" : "Login Page."
  }, {
    "name" : "parentSuite",
    "value" : "tests"
  }, {
    "name" : "suite",
    "value" : "login_page_test"
  }, {
    "name" : "subSuite",
    "value" : "TestLoginPage"
  }, {
    "name" : "host",
    "value" : "fv-az891-551"
  }, {
    "name" : "thread",
    "value" : "2367-MainThread"
  }, {
    "name" : "framework",
    "value" : "pytest"
  }, {
    "name" : "language",
    "value" : "cpython3"
  }, {
    "name" : "package",
    "value" : "tests.login_page_test"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ ],
  "links" : [ ],
  "hidden" : false,
  "retry" : false,
  "extra" : {
    "severity" : "normal",
    "retries" : [ ],
    "categories" : [ {
      "name" : "Test defects",
      "matchedStatuses" : [ ],
      "flaky" : false
    } ],
    "history" : {
      "statistic" : {
        "failed" : 0,
        "broken" : 4,
        "skipped" : 0,
        "passed" : 57,
        "unknown" : 0,
        "total" : 61
      },
      "items" : [ {
        "uid" : "230c1022e514950",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/60//#testresult/230c1022e514950",
        "status" : "broken",
        "statusDetails" : "selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome=129.0.6668.89)\nStacktrace:\n#0 0x559b3105502a <unknown>\n#1 0x559b30d3b5e0 <unknown>\n#2 0x559b30d337eb <unknown>\n#3 0x559b30d23a09 <unknown>\n#4 0x559b30d25744 <unknown>\n#5 0x559b30d23cbd <unknown>\n#6 0x559b30d2352f <unknown>\n#7 0x559b30d23422 <unknown>\n#8 0x559b30d21493 <unknown>\n#9 0x559b30d21ada <unknown>\n#10 0x559b30d3df41 <unknown>\n#11 0x559b30dcf8c5 <unknown>\n#12 0x559b30dafb22 <unknown>\n#13 0x559b30dced7d <unknown>\n#14 0x559b30daf8c3 <unknown>\n#15 0x559b30d7d6b3 <unknown>\n#16 0x559b30d7e68e <unknown>\n#17 0x559b3101fa2b <unknown>\n#18 0x559b310239b1 <unknown>\n#19 0x559b3100c225 <unknown>\n#20 0x559b31024532 <unknown>\n#21 0x559b30ff138f <unknown>\n#22 0x559b31043f28 <unknown>\n#23 0x559b310440f3 <unknown>\n#24 0x559b31053e7c <unknown>\n#25 0x7fc2a6494ac3 <unknown>",
        "time" : {
          "start" : 1728697287687,
          "stop" : 1728697287687,
          "duration" : 0
        }
      }, {
        "uid" : "7f147ddc9d5e2d4b",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/59//#testresult/7f147ddc9d5e2d4b",
        "status" : "broken",
        "statusDetails" : "selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome=129.0.6668.89)\nStacktrace:\n#0 0x55fbb8b6702a <unknown>\n#1 0x55fbb884d5e0 <unknown>\n#2 0x55fbb88457eb <unknown>\n#3 0x55fbb8835a09 <unknown>\n#4 0x55fbb8837744 <unknown>\n#5 0x55fbb8835cbd <unknown>\n#6 0x55fbb883552f <unknown>\n#7 0x55fbb8835422 <unknown>\n#8 0x55fbb8833493 <unknown>\n#9 0x55fbb8833ada <unknown>\n#10 0x55fbb884ff41 <unknown>\n#11 0x55fbb88e18c5 <unknown>\n#12 0x55fbb88c1b22 <unknown>\n#13 0x55fbb88e0d7d <unknown>\n#14 0x55fbb88c18c3 <unknown>\n#15 0x55fbb888f6b3 <unknown>\n#16 0x55fbb889068e <unknown>\n#17 0x55fbb8b31a2b <unknown>\n#18 0x55fbb8b359b1 <unknown>\n#19 0x55fbb8b1e225 <unknown>\n#20 0x55fbb8b36532 <unknown>\n#21 0x55fbb8b0338f <unknown>\n#22 0x55fbb8b55f28 <unknown>\n#23 0x55fbb8b560f3 <unknown>\n#24 0x55fbb8b65e7c <unknown>\n#25 0x7feafa894ac3 <unknown>",
        "time" : {
          "start" : 1728610978893,
          "stop" : 1728610978893,
          "duration" : 0
        }
      }, {
        "uid" : "25a53db6ce759ce3",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/58//#testresult/25a53db6ce759ce3",
        "status" : "passed",
        "time" : {
          "start" : 1728524843969,
          "stop" : 1728524843972,
          "duration" : 3
        }
      }, {
        "uid" : "88ee2558aa5a3d8c",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/57//#testresult/88ee2558aa5a3d8c",
        "status" : "passed",
        "time" : {
          "start" : 1728438517775,
          "stop" : 1728438517777,
          "duration" : 2
        }
      }, {
        "uid" : "9d07813f66739a85",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/56//#testresult/9d07813f66739a85",
        "status" : "passed",
        "time" : {
          "start" : 1728352092077,
          "stop" : 1728352092079,
          "duration" : 2
        }
      }, {
        "uid" : "b0f5c4f985f40953",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/55//#testresult/b0f5c4f985f40953",
        "status" : "broken",
        "statusDetails" : "selenium.common.exceptions.TimeoutException: Message: Can't find element by locator https://brainup.site/groups",
        "time" : {
          "start" : 1728266300951,
          "stop" : 1728266300951,
          "duration" : 0
        }
      }, {
        "uid" : "b101f702cc8a0464",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/54//#testresult/b101f702cc8a0464",
        "status" : "passed",
        "time" : {
          "start" : 1728179637405,
          "stop" : 1728179637408,
          "duration" : 3
        }
      }, {
        "uid" : "96e0f8df90f1cbd8",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/53//#testresult/96e0f8df90f1cbd8",
        "status" : "passed",
        "time" : {
          "start" : 1728092853379,
          "stop" : 1728092853382,
          "duration" : 3
        }
      }, {
        "uid" : "58563e69b79a8ea6",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/52//#testresult/58563e69b79a8ea6",
        "status" : "passed",
        "time" : {
          "start" : 1728006556769,
          "stop" : 1728006556771,
          "duration" : 2
        }
      }, {
        "uid" : "c4d46e0ba757d511",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/51//#testresult/c4d46e0ba757d511",
        "status" : "passed",
        "time" : {
          "start" : 1727920188504,
          "stop" : 1727920188507,
          "duration" : 3
        }
      }, {
        "uid" : "2d8c8acaa0ceb7e3",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/50//#testresult/2d8c8acaa0ceb7e3",
        "status" : "passed",
        "time" : {
          "start" : 1727833649747,
          "stop" : 1727833649750,
          "duration" : 3
        }
      }, {
        "uid" : "4f4d1673e832eab5",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/49//#testresult/4f4d1673e832eab5",
        "status" : "passed",
        "time" : {
          "start" : 1727747773490,
          "stop" : 1727747773493,
          "duration" : 3
        }
      }, {
        "uid" : "57a56f533ee5e436",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/48//#testresult/57a56f533ee5e436",
        "status" : "passed",
        "time" : {
          "start" : 1727661057849,
          "stop" : 1727661057851,
          "duration" : 2
        }
      }, {
        "uid" : "6246043e2f05d749",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/47//#testresult/6246043e2f05d749",
        "status" : "passed",
        "time" : {
          "start" : 1727574878789,
          "stop" : 1727574878791,
          "duration" : 2
        }
      }, {
        "uid" : "850741f041a00581",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/46//#testresult/850741f041a00581",
        "status" : "passed",
        "time" : {
          "start" : 1727488107304,
          "stop" : 1727488107307,
          "duration" : 3
        }
      }, {
        "uid" : "d7f70d31716e3eb1",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/45//#testresult/d7f70d31716e3eb1",
        "status" : "passed",
        "time" : {
          "start" : 1727401807215,
          "stop" : 1727401807217,
          "duration" : 2
        }
      }, {
        "uid" : "11fb0d1826193211",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/44//#testresult/11fb0d1826193211",
        "status" : "passed",
        "time" : {
          "start" : 1727315292974,
          "stop" : 1727315292977,
          "duration" : 3
        }
      }, {
        "uid" : "2e33a8ecab1a6c2a",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/43//#testresult/2e33a8ecab1a6c2a",
        "status" : "passed",
        "time" : {
          "start" : 1727228948466,
          "stop" : 1727228948469,
          "duration" : 3
        }
      }, {
        "uid" : "9e6ead6d950af7df",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/42//#testresult/9e6ead6d950af7df",
        "status" : "passed",
        "time" : {
          "start" : 1727142496227,
          "stop" : 1727142496229,
          "duration" : 2
        }
      }, {
        "uid" : "74009bf35d5e39f5",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/41//#testresult/74009bf35d5e39f5",
        "status" : "passed",
        "time" : {
          "start" : 1727056027698,
          "stop" : 1727056027701,
          "duration" : 3
        }
      } ]
    },
    "tags" : [ ]
  },
  "source" : "3a3aa6847e4962f5.json",
  "parameterValues" : [ ]
}